toqito/matrices/cyclic_permutation.py

```diff
@@ -59,4 +56,5 @@ def cyclic_permutation_matrix(n: int, k: int = 1) -> np.ndarray:
     p_mat[0, -1] = 1
 
     result_mat = np.linalg.matrix_power(p_mat, k)
+    np.set_printoptions(edgeitems=n*n, linewidth=40)
```

Comment:
Maybe you can get rid of this line. If the issue was related to using `print(cyclic_permutation_matrix(n, k))` vs `cyclic_permutation_matrix(n, k)`, then this addition is not required as you have corrected these lines in the example.

Reply:
I'm sure I'll be taking on some more!